docs/modules/ROOT/pages/advanced-how-tos/managing-multiple-versions.adoc

```diff
@@ -161,6 +161,27 @@ event == "push" && target_branch == "v1.0.0"
 
 * The `appstudio.openshift.io/application` and `appstudio.openshift.io/component` labels must be adjusted to specify the right Application and Component respectively. Failing to do this will cause builds of the pipeline to be associated with the wrong application or component.
 
+=== Create ImageRepositories for each component
+
+This must be done for each component to authenticate with the output-registry when pushing built images.
+
+[source,yaml]
+.imagerepository.yaml
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ImageRepository
+metadata:
+  labels:
+    appstudio.redhat.com/application: konflux-multibranch-sample-v1-0-0
+    appstudio.redhat.com/component: konflux-multibranch-sample-v1-0-0
+  name: imagerepository-for-konflux-multibranch-sample-v1-0-0-konflux-multibranch-sample-v1-0-0
+  namespace: konflux-multibranch-sample-template
+spec:
+  image:
+    name: konflux-multibranch-sample-v1-0-0/konflux-multibranch-sample-v1-0-0
+    visibility: public
```

Comment:
Can we note that this value can be private as well so that private repos can be onboarded with private images?

Reply:
Added